README.rst

```diff
@@ -304,7 +304,7 @@ NI-SWITCH Python API Status
 +-------------------------------+-------------------------+
 | NI-SWITCH (niswitch)          |                         |
 +===============================+=========================+
-| Driver Version Tested Against | 17.0.0                  |
+| Driver Version Tested Against | 18.0.0                  |
```

Comment:
Yes! nimi-bot has been upgraded to the versions listed there.

Otherwise the new functions (import / export) would not work during the system tests.